x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -46,46 +55,289 @@ var security = (function () {
         [0x04000000, 'PARTIAL_SECRETS_ACCOUNT'],
     ];
 
+    // Kerberos TGT and TGS Ticket Options
+    // https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4768
+    // https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4769
+    var ticketOptions = [
+        "Reserved",
+        "Forwardable",
+        "Forwarded",
+        "Proxiable",
+        "Proxy",
+        "Allow-postdate",
+        "Postdated",
+        "Invalid",
+        "Renewable",
+        "Initial",
+        "Pre-authent",
+        "Opt-hardware-auth",
+        "Transited-policy-checked",
+        "Ok-as-delegate",
+        "Request-anonymous",
+        "Name-canonicalize",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Disable-transited-check",
+        "Renewable-ok",
+        "Enc-tkt-in-skey",
+        "Unused",
+        "Renew",
+        "Validate"];
+
+    // Kerberos Encryption Types
+    // https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4768
+    // https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4768
+    var ticketEncryptionTypes = {
+        "0x1": "DES-CBC-CRC",
+        "0x3": "DES-CBC-MD5",
+        "0x11": "AES128-CTS-HMAC-SHA1-96",
+        "0x12": "AES256-CTS-HMAC-SHA1-96",
+        "0x17": "RC4-HMAC",
+        "0x18": "RC4-HMAC-EXP",
+        "0xFFFFFFFF": "FAIL",
+        "0xffffffff": "FAIL",
```

Comment:
What do you think about normalizing the hex string so it is all upper or all lowercase?  That way we won't have to duplicate entries.

Reply:
normalized. Only the lower case entry was left